### PR TITLE
Feature/spinners

### PR DESCRIPTION
**Pull request by CesarDav**
### Describe the work done

refactoring change of password

### Definition of Done

- [ ] User Story compliant
- [ ] Feature completed
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged
- [ ] Product Owner validated

### CODE REVIEW

File: src/modules/auth/ChangePasswordView.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+      email: '',
+      code: getUrlParam(this.props.location.search, 'oobCode'),
+    };
+  }
+
+  onSubmit = (e) => {
+    e.preventDefault();
+    const { firstInput, secondInput, code, email } = this.state;
+
+    if (firstInput === secondInput) {
```
Comment (alacret) [inline src/modules/auth/ChangePasswordView.js]: @CesarDav utiliza aqui la tecnica the `soon return` significando que, primero escapa los escenarios de errores y despues haces la logica principal.
En otras palabras:
En vez de esto:
```
if (condition){
    // una logica
    // muy complicada
    // y muy larga
}else{
    // return o throw o dispatch
}
```

Haz esto:
```
if (!condition)
    // return o throw o dispatch

// una logica
// muy complicada
// y muy larga
```

File: src/modules/auth/ChangePasswordView.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+      let newPassword = firstInput;
+      if (newPassword.length >= 6) {
```
Comment (alacret) [inline src/modules/auth/ChangePasswordView.js]: lo mismo aqui

File: src/modules/auth/auth-actions.js [hunk 3]
```diff
@@ -199,3 +196,16 @@ export const requestPasswordReset = async (email) => {
     .then((send) => Flux.dispatchEvent(REQUEST_RECOVER_PASSWORD, send))
     .catch((err) => Flux.dispatchEvent(USER_ERROR, err));
 };
+
+//  update property
+export const updatePropertyDoc = async (email) => {
```
Comment (alacret) [inline src/modules/auth/auth-actions.js]: por que esto no se hace en la funcion confirmPassword?

**Comment by CesarDav** [comment]
revisa broo @alacret

**Pull request merged**